tests/integration/sandbox.py

```diff
@@ -130,9 +133,10 @@ def init(self):
         LOG.info('Cluster created: %s', dbid)
         return self
 
-    def join(self, addresses):
+    def join(self, addresses, expect_error=None):
         self.start()
-        self.cluster('join', *addresses)
+        logging.info("In Join")
```

Comment:
Is this really needed or a leftover from earlier tests?

Reply:
it's an artifact of me putting in logging to understand what was going on, need to remove.